example/src/App.js

```diff
@@ -280,6 +282,30 @@ const App = () => {
           With actions and onDelete
         </EasyPill>
       </section>
+      <section>
+        <h1>Collapse</h1>
+        <Collapse isOpen>Collapse content</Collapse>
+      </section>
+      <section>
+        <h1>ExpansionPanel</h1>
+        <ExpansionPanel defaultExpanded={false}>
+          <ExpansionPanel.Header>Click me!</ExpansionPanel.Header>
+          <ExpansionPanel.Body>├── Body</ExpansionPanel.Body>
+        </ExpansionPanel>
+      </section>
+      <section>
+        <h1>Accordion</h1>
+        <Accordion>
```

Comment:
You didn't import `Accordion`. :(

Reply:
https://github.com/cision/rover-ui/pull/104